book/src/proposals/booting-from-snapshot.md

```diff
@@ -0,0 +1,21 @@
+# Booting From a Snapshot
+
+## Problem
+
+A validator booting from a snapshot should know that they are caught up to the cluster before voting.
+
+## Solution
+
+A booting validator needs to have some set of trusted validators whose votes it will rely upon to ensure that it has caught up to the cluster and has a valid snapshot. With that set, the validator can
+
+1. Get a snapshot to boot from.
```

Comment:
Combined with the sentence above, this makes it sound like the snapshot must come from the set of trusted validators.

Reply:
If I don't know who to trust or just have no friends, I could just default 2/3rds of the cluster for my trusted validator set and follow the remaining steps in this procedure too.

Reply:
The problem with this is that you could be fed a fake cluster with fake validators and stake, and without trusted pubkeys you can't tell the difference. If that cluster claims to be at slot 100,000 while the real cluster is at 10,000, if you vote once on the fake cluster, you're locked out of the real cluster for 90,000 slots.

Reply:
Right, sorry my point was more that observing your canary transactions land on the chain you think you're on (regardless of who you want to trust on on that chain) seem like a good thing to do before submitting a real vote to that chain.